pom.xml

```diff
@@ -24,7 +24,7 @@
 
   <properties>
     <java.version>11</java.version>
-    <folio-spring-base.version>4.0.0</folio-spring-base.version>
+    <folio-spring-base.version>4.1.0</folio-spring-base.version>
```

Comment:
folio-spring-base 4.1.0 is based on spring-boot-starter-parent 2.7, therefore spring-boot-starter-parent should be bumped from 2.6.4 to 2.7.1

Reply:
Thank you. Fixed.